setup.cfg

```diff
@@ -22,6 +22,8 @@ include_package_data = true
 
 install_requires =
     attrs
+    cachecontrol
+    lockfile
```

Comment:
`cachecontrol[filecache]` has the right deps

Reply:
done